src/test/mediapipe/python/scripts/bad_execute_import_error.py

```diff
@@ -0,0 +1,27 @@
+#*****************************************************************************
+# Copyright 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#*****************************************************************************
+
+class OvmsPythonModel:
+
+    def initialize(self, kwargs: dict):
+        return True
+
+    def execute(self, inputs: list) -> list:
+        import non_existing_module
```

Comment:
Should we check what happens when the import is in line 16? Outside of class? We should have an error in different time

Reply:
It could be one of the cases for initialization import errors. If it happens outside of the class it should still fail during initialization phase.

Reply:
Please add such test case